### PR TITLE
cv styled (aleixcam)

### PR DESCRIPTION
**Pull request by aleixcam**

### CODE REVIEW

File: staff/aleix-caminal/app-template/about.html [hunk 1]
```diff
@@ -27,8 +37,5 @@ <h2>Some of our clients</h2>
                 </ul>
```
Comment (pepdbm7) [inline staff/aleix-caminal/app-template/about.html:37]: forgot to delete it maybe

File: staff/aleix-caminal/cv-styled/assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  transition: all 0.4s;
+}
+.button:hover {
+  color: #fff;
+  background-color: #0960a5;
+  transition: all 0.2s;
+}
+
+.text--mb {
+  margin-bottom: 0.25rem !important;
```
Comment (seand52) [inline staff/aleix-caminal/cv-styled/assets/css/style.css:55]: Normally !important is used as a last resort. Is there no other way of ensuring this style will be applied?

Comment (aleixcam) [reply staff/aleix-caminal/cv-styled/assets/css/style.css:55]: there's really no other way to do it :/

File: staff/aleix-caminal/app-template/about.html (hunk 1 above)
Comment (aleixcam) [reply staff/aleix-caminal/app-template/about.html:37]: the missing part is inside a collapsible section in github, may you have missed it

File: staff/aleix-caminal/cv-styled/assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,172 @@
+body {
+  display: grid;
+  grid-template: auto auto/1fr 260px;
+  grid-gap: 1.25rem;
+  padding: 1.25rem;
+  font-family: TT Commons, system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Droid Sans, Helvetica Neue;
```
Comment (diesttro) [inline staff/aleix-caminal/cv-styled/assets/css/style.css:6]: Too much fonts were specified.

You need to specify at least the base font type: sans or sans-serif. This charge a font of this type if all the others fail.

Comment (aleixcam) [reply staff/aleix-caminal/cv-styled/assets/css/style.css:6]: true xD

**Pull request merged**